alphapept/gui/start.py

```diff
@@ -65,3 +66,11 @@ def start():
             "https://datashare.biochem.mpg.de/s/p8Qu3KolzbSiCHH/download",
         )
         st.write("Put both files in one folder and go to the New experiment tab.")
+
+        
+    latest_version = get_latest_version()
+    if latest_version and VERSION_NO != latest_version:
+        st.info(
+            "You're using alphapept {0} but version {1} is now avaliable!. See [here](https://github.com/MannLabs/alphapept) for details".format(
```

Comment:
Python now can do [f-strings](https://www.python.org/dev/peps/pep-0498/), which can be very useful.

With f_strings this would look like this:

`f"You're using alphapept {VERSION_NO} but version {latest_version} is now avaliable!. See [here](https://github.com/MannLabs/alphapept) for details"
`

Reply:
Oh cool! That's a much cleaner syntax!